lib/improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -467,9 +486,9 @@ def calculate_wet_bulb_temperature(self, temperature, relative_humidity,
 
             # Recalculate the saturation mixing ratio
```

Comment:
I know this is in the original but is there a reason that the saturation mixing ratio is calculated at the end? Especially when it isn't returned?

Reply:
No idea.

Reply:
This is happening inside the `while` loop as part of the iteration to find the wet bulb temperature.  Saturation mixing ration is updated in each iteration.

Reply:
Ok that makes sense now, thanks @cgsandford